node/lib/openshift-origin-node/model/application_container_ext/environment.rb

```diff
@@ -273,13 +272,18 @@ def validate_ssh_keys(ssh_keys)
 
         # Add user environment variable(s)
         def user_var_add(variables, gears = [])
+          config = ::OpenShift::Config.new
```

Comment:
We should add `require 'openshift-origin-common'` if we're going to use `OpenShift::Config`.